assets/data/cards_pt-br.json

```
@@ -482,6 +482,13 @@
         "Conceito",
         "Back-end"
       ]
+    },
+    {
+      "title": "Callback",
+      "description": "Uma callback é uma função que é passada como argumento para outra função e é executada após algum evento. É uma forma de garantir que uma função só será executada após a conclusão de outra.",
```

Tenho uma sugestão, veja o que acha. Essa explicação usa uma analogia simples para tornar o conceito mais compreensível.

```suggestion
      "description": "Uma callback é uma função que você passa para outra função como um argumento para que ela seja chamada depois que a função principal terminar o que está fazendo. É como pedir a alguém para te lembrar de algo depois que eles terminarem uma tarefa. Isso ajuda a garantir que uma função só seja executada depois que outra função já fez o seu trabalho.",
```